sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceAsyncClientTest.java

```diff
@@ -87,6 +86,51 @@ public void serviceCreateTable() {
             .verify();
     }
 
+
+    @Test
+    public void serviceAsyncClientCreateTableFailsWithIllegalArgumentExceptionIfTableNameInURI() {
+        // Arrange
+        String assertionString = "Table name found within client endpoint URI. Remove table name from endpoint.";
+        String tableName = testResourceNamer.randomName("test", 20);
+        final ClientSecretCredential credential = new ClientSecretCredentialBuilder()
+            .clientId(Configuration.getGlobalConfiguration().get("TABLES_CLIENT_ID", "clientId"))
+            .clientSecret(Configuration.getGlobalConfiguration().get("TABLES_CLIENT_SECRET", "clientSecret"))
+            .tenantId(testResourceNamer.randomUuid())
+            .build();
+
+        TableServiceAsyncClient malformedClient = getClientBuilder(Configuration.getGlobalConfiguration().get(
+            "TABLES_ENDPOINT","https://tablestests.table.core.windows.com")+"/"+tableName,credential,true)
```

Comment:
nit; CheckStyle is going to complain about this.
```suggestion
            "TABLES_ENDPOINT", "https://tablestests.table.core.windows.com") + "/" + tableName, credential, true)
```

Reply:
There are a few other tests I saw do this on this PR, so this comment applies to all those too.